apps/www/_blog/2025-09-30-postgrest-13-release.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: PostgREST 13
+description: New features and changes in PostgREST version 13.
+author: steve_chavez,laurenceisla,avallete
+image: 2025-09-30-postgrest-13-release/postgrest-13-release.png
+thumb: 2025-09-30-postgrest-13-release/postgrest-13-release.png
+categories:
+  - postgres
+tags:
+  - postgres
+  - postgrest
+  - release-notes
+date: '2025-09-30'
+---
+
+PostgREST 13 is out! It comes with API and Observabilty improvements. In this post, we'll see what's new.
+
+## Spread To-Many relationships
+
+This new feature allows you to represent one-to-many and many-to-many relationships as flat JSON arrays.
+
+For example, if you have database similar to IMDB and you’d like to represent it as a hierarchical JSON structure for your frontend, like so:
+
+```json
+[
+  {
+    "title": "The Shawshank Redemption",
+    "actors": ["Tim Robbins", "Morgan Freeman"],
+    "genres": ["Drama"]
+  },
+  {
+    "title": "The Godfather",
+    "actors": ["Marlon Brando", "Al Pacino"],
+    "genres": ["Drama", "Crime"]
+  },
+  {
+    "title": "The Dark Knight",
+    "actors": ["Christian Bale", "Heath Ledger"],
+    "genres": ["Drama", "Crime", "Action"]
+  }
+]
+```
+
+You can now do it this way:
+
+<$CodeTabs>
+
+```js name=JavaScript
+const { data, error } = await supabase.from('titles').select(`
+  title:primary_title,
+  ...people(actors:primary_name),
+  ...genres(genres:name)
+`)
+```
+
+```bash name=Bash
+curl 'https://<your-domain>/titles'
+  -d "select=title:primary_title,...people(actors:primary_name),...genres(genres:name)"
```

Comment:
Adding `-d` turns it into a `POST` request, it needs `--get` to make it work as a query-string.

```suggestion
curl --get 'https://<your-domain>/titles'
  -d "select=title:primary_title,...people(actors:primary_name),...genres(genres:name)"
```